spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+RSpec.describe "Project Show" do
+
+  describe "User Story 1: As a visitor," do 
+    describe "When I visit a project's show page ('/projects/:id')" do 
+      it "it has project's name and material" do
+        #Challenge
+        recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+        #Project
+        news_chic = recycled_material_challenge.projects.
+        create(name: "News Chic", material: "Newspaper")
+
+        visit "/projects/#{news_chic.id}"
+
+        expect(page).to have_content("News Chic")
+        expect(page).to have_content("Newspaper")
+      end
+
+      it "it also has the theme of the challenge that this project belongs to" do
+        #Challenge
+        recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+        #Project
+        news_chic = recycled_material_challenge.projects.
+        create(name: "News Chic", material: "Newspaper")
+
+        visit "/projects/#{news_chic.id}"
+        
+        expect(page).to have_content("Recycled Material")
+      end
+    end
+  end
+
+  describe "User Story 3: As a visitor," do
+   describe "When I visit a project's show page" do
+      recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
```

Comment:
Instead of having this test data out in a describe block, Id suggest using a `before :each` in order to make sure the test data will be cleaned up after each case. Sometimes things can get funky if the data is outside of an `it` block or a `before :each` so it's best to keep them in those